DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/DelegateGroups/AddDelegateGroup.cshtml

```diff
@@ -0,0 +1,62 @@
+﻿@inject IConfiguration Configuration;
+@using DigitalLearningSolutions.Web.Models.Enums
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.DelegateGroups
+@using Microsoft.AspNetCore.Mvc.TagHelpers

```

Comment:
I think the TagHelpers is unneeded.

Reply:
Hm, I wonder how that got added in. Nice catch!